src/pages/profile/profile.scss

```diff
@@ -18,5 +18,69 @@ page-profile {
     .name {
         font-size: 30px;
     }
+    .section {
+        margin-top: 20px;
+    }
+    .label, user-info {
+        font-size: 20px;
+        color: black;
+         overflow: hidden; 
+         text-overflow: clip;
+         padding: 10px;
+    }
+    .label {
+        margin: auto;
+        min-width: 100px;
+    }
+
+    .user-info {
+        margin-left: 20%;
+        position: relative;
+    }
+    .user-info-colum {
+       width: 80%; 
+       margin: auto;
+       max-width: 500px;
+    }
+    .buttons {
+       
+        //display: block;
+       // margin: 10px;
+    } 
+    .buttons-div {
+        display: flex;
+        justify-content: center;
+    }
+    .buttons1 {
+        width: 100px;
+        margin: 10px 10px auto 10px;
+    }
+   
+    @media screen and (min-width: 768px) {
```

Comment:
Your styling formatting and media query layout is great, can you discuss your methodology in class during CR?

Reply:
sure!